composer/metrics/metrics.py

```diff
@@ -55,7 +55,9 @@ def update(self, logits: Tensor, targets: Tensor):
 
     def compute(self):
         """Aggregate state across all processes and compute final metric."""
-        return 100 * (self.total_intersect / self.total_union).mean()  #type: ignore (third-party)
+        per_class_miou = self.total_intersect / self.total_union  # type: ignore (third-party)
+        per_class_miou[self.total_union == 0] = 0
+        return 100 * per_class_miou.mean()
```

Comment:
Should we be excluding classes that aren't present instead of 0ing it out? CC: @Landanjs @coryMosaicML

Reply:
Better to exclude, I think. What do you think @Landanjs?

Reply:
I think excluding makes more sense as well. Otherwise, the mIoU will be very small depending on the number of classes and how many are present.